Yoast/wordpress-seo
# Make the add-ons tab non interactive when only one add-on is enabled

## Summary

This PR can be summarized in the following changelog entry:

* Improves accessibility of the add-ons tabs in the meta box.

## Relevant technical choices:
When there's only one add-on enabled, its tab in the meta box shouldn't be a link that does nothing.

Thanks to @andizer assistance and help we've explored a PHP way to output a different markup when there's only one add-on enabled. Decided against it for backwards compatibility issues.

I've opted for resorting back to a couple lines of jQuery. This PR does the following:

- when there's only one add-on tab, changes the link to a non-interactive `<span>` element
- fixes an apparently long standing issue, where the add-ons tab content was outside of a wrapper div element, while all the other tabs print out the content _inside_ this div
- adds a docblock thus reducing the ESLint max-warnings

## Test instructions
- build
- activate both News SEO and Video SEO
- check the metabox > add-ons
- check both the News and Video tabs are links and can be clicked
- deactivate one of the add-ons
- check again the metabox > add-ons
- there's now only one tab: check it's not clickable and there are no visual differences

## UI changes

Screenshot: this tab should not be clickable:

<img width="529" alt="screenshot 2019-01-23 at 14 36 32" src="https://user-images.githubusercontent.com/1682452/51610176-6a088780-1f1c-11e9-953e-26868a1e3d15.png">

Fixes #5919
